src/test/functional/pages/PimPage.ts

```diff
@@ -1,6 +1,42 @@
 import { BasePage } from './BasePage';
+import { newEmployeeTestData } from '../data';
 
 export class PimPage extends BasePage {
+    protected readonly sharePhotosButton = this.page.getByRole('button', { name: 'Share Photos' })
+    protected readonly addEmployeeButton = this.page.getByRole('button', { name: ' Add' })
+    protected readonly firstNameInput = this.page.getByPlaceholder('First Name')
+    protected readonly middleNameInput = this.page.getByPlaceholder('Middle Name')
+    protected readonly lastNameInput = this.page.getByPlaceholder('Last Name')
+    protected readonly saveUserButton = this.page.getByRole('button', { name: 'Save' })
+    public readonly confirmDeleteButton = this.page.getByRole('button', { name: 'Yes, Delete' })
 
+    public async randomNewEmplyeeNameText(randomNewEmpoyeeName: string) {
+      const element = `.oxd-table-cell:has(:text("${randomNewEmpoyeeName}"))`;
+      return element;
+    }
 
+    public async locateTrashBinByRandomEmployeeName(randomNewEmpoyeeName: string) {
+      const element = `.oxd-table-card:has(:text("${randomNewEmpoyeeName}")) .oxd-icon.bi-trash`;
```

Comment:
In all those similiar functions I think it's better to just do `return `.oxd-table-card:has(:text("${randomNewEmpoyeeName}")) .oxd-icon.bi-trash``
Instead of creating an variable that will just be returned. But this is just my personal preference.

Reply:
agree

Reply:
I do not know why but when I return it directly then tests are unstable and failed, so I left it as it is any suggestion why it could be like this?